numpy/add_newdocs.py

```diff
@@ -463,6 +463,50 @@ def luf(lamdaexpr, *args, **kwargs):
 
     """))
 
+add_newdoc('numpy.core', 'nested_iters',
+    """
+    Create a outermost-first tuple of iterators. They are nested so that
+    advancing one will change the inner iterators to point at the new
+    element in the outer iterator.
```

Comment:
I feel this is still a bit unclear, especially the phases "outermost-first" or "new element". Maybe would be good to use the word `axes`, something like

```
Create a tuple of `nditer`s which each iterate over different axes of an array. They are nested such that advancing the earlier iterators in the tuple will reset the position of later iterators to the beginning of the row specified by the earlier iterator.
```
But that still feels a bit muddled...